feature-detects/elem/picture.js

```diff
@@ -6,7 +6,7 @@
   "authors": ["Scott Jehl", "Mat Marquis"],
   "notes": [{
     "name": "Specification",
-    "href": "http://picture.responsiveimages.org"
+    "href": "http://w3c.github.io/html/semantics-embedded-content.html#the-picture-element"
```

Comment:
👎 on this change. The w3c html spec is dead, so we should not be linking to it for documentation. If you are against linking to the RIWG site as it is a redirect, then please update it to the site it is redirecting to

Reply:
will change to the redirected page...